integration/setup/tenant_cluster.go

```diff
@@ -192,45 +236,7 @@ func ensureHostVPC(ctx context.Context, config Config) (string, error) {
 	return vpcID, nil
 }
 
-func InstallAWSConfig(ctx context.Context, id string, config Config) error {
-	vpcID, err := ensureHostVPC(ctx, config)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
-	var values string
-	{
-		c := chartvalues.APIExtensionsAWSConfigE2EConfig{
-			CommonDomain:         env.CommonDomain(),
-			ClusterName:          id,
-			SSHPublicKey:         env.IDRSAPub(),
-			VersionBundleVersion: env.VersionBundleVersion(),
-
-			AWS: chartvalues.APIExtensionsAWSConfigE2EConfigAWS{
-				Region:            env.AWSRegion(),
-				APIHostedZone:     env.AWSAPIHostedZoneGuest(),
-				IngressHostedZone: env.AWSIngressHostedZoneGuest(),
-				RouteTable0:       env.AWSRouteTable0(),
-				RouteTable1:       env.AWSRouteTable1(),
-				VPCPeerID:         vpcID,
-			},
-		}
-
-		values, err = chartvalues.NewAPIExtensionsAWSConfigE2E(c)
-		if err != nil {
-			return microerror.Mask(err)
-		}
-	}
-
-	err = config.Release.Install(ctx, key.AWSConfigReleaseName(id), release.NewStableChartInfo("apiextensions-aws-config-e2e-chart"), values, crExistsCondition(ctx, id, config))
```

Comment:
This went up. The only change is `config.Release.Install` replaced with `config.Release.EnsureInstalled`.